tarpc/tests/compile_fail/serde_transport/must_use_tcp_connect.stderr

```diff
@@ -1,4 +1,4 @@
-error: unused `Connect` that must be used
+error: unused `tarpc::serde_transport::tcp::Connect` that must be used
```

Comment:
My thinking is because prior to this PR there was only one `Connect` type (`tarpc::serde_transport::tcp::Connect`) but now there are two (`tarpc::serde_transport::unix::Connect`, so Rust is giving the full path to disambiguate?